glotaran/project/project.py

```diff
@@ -674,6 +697,10 @@ def optimize(
             The maximum number of function evaluations.
         clp_link_tolerance : float
             The CLP link tolerance.
+        data_lookup_overwrite: Mapping[str, LoadableDataset] | None
```

Comment:
```suggestion
        data_lookup_override: Mapping[str, LoadableDataset] | None
```

Reply:
Fixed in [5e7a47d](https://github.com/glotaran/pyglotaran/pull/1257/commits/5e7a47de52900b22acde581c9e934280d5e95975)

Reply:
Fixed in [5e7a47d](https://github.com/glotaran/pyglotaran/pull/1257/commits/5e7a47de52900b22acde581c9e934280d5e95975)